docs/topics/rn-new-features.adoc

```diff
@@ -7,25 +7,25 @@
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
+[IMPORTANT]
+====
+Rules for migrating from Java EE8 to Jakarta EE9 and OpenRewrite support are both provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
 .Rules for migrating from Java EE8 to Jakarta EE9
 {ProductShortName} rules replace Java EE8 artifact dependencies and package import statements and rename XML schema namespaces, prefixed properties, and bootstrapping files for Jakarta EE9.
 
-Most rules contain quick fixes to automate source code changes when using an {ProductShortName} IDE plugin or extension for Eclipse, Eclipse Che, IntelliJ IDEA, Microsoft Visual Studio Code, or Red Hat CodeReady Studio.
+You run these rules by specifying the `jakarta-ee` target.
 
-To execute the rules, use the target `jakarta-ee`.
+Most rules contain quick fixes to automate source code changes when using an {ProductShortName} IDE plugin or extension for Eclipse, Eclipse Che, IntelliJ IDEA, Microsoft Visual Studio Code, or Red Hat CodeReady Studio.
 
 .OpenRewrite recipe support
 
-link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. You can execute OpenRewrite recipes by using the link:{ProductDocUserGuideURL}[{ProductShortName} CLI].
-
-The first OpenRewrite recipe, which is shipped with {ProductShortName} 5.2.1, renames imported `javax` packages to their `jakarta` equivalents. 
+link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. You can run OpenRewrite recipes by using the link:{ProductDocUserGuideURL}[{ProductShortName} CLI].
 
-[IMPORTANT]
-====
-OpenRewrite recipe support is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-
-See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
-====
+`org.jboss.windup.JavaxToJakarta`, which is shipped with {ProductShortName} 5.2.1, renames imported `javax` packages to their `jakarta` equivalents.
```

Comment:
The `org.jboss.windup.JavaxToJakarta` OpenRewrite recipe renames imported `javax` packages to their `jakarta` equivalents.

I think adding "The .. recipe" makes it clearer what this new entity is.
You don't need "which is shipped with" because this is a release note, so it's obvious that it comes with this version.